README.rst

```diff
@@ -747,6 +747,18 @@ so an example nginx configuration might look like::
       }
   }
 
+and an example apache configuration may look like::
+
+    <VirtualHost *:443>
+        SSLEngine on
+        ServerName matrix.example.com;
+
+        <Location /_matrix>
+            ProxyPass http://127.0.0.1:8008/_matrix nocanon
+            ProxyPassReverse http://127.0.0.1:8008/_matrix
```

Comment:
this should probably include something to set the X-Forwarded-For header ?

Reply:
Isn't that set in homeserver.yaml?

Reply:
the point is to get apache to set the header, so that synapse can use it.

Reply:
Gotcha. I'll add it this afternoon.